mvc/src/test/java/com/interface21/webmvc/servlet/mvc/tobe/HandlerExecutionTest.java

```diff
@@ -0,0 +1,53 @@
+package com.interface21.webmvc.servlet.mvc.tobe;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+
+import com.interface21.context.stereotype.Controller;
+import com.interface21.webmvc.servlet.ModelAndView;
+import com.interface21.webmvc.servlet.view.JsonView;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Set;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.reflections.Reflections;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+class HandlerExecutionTest {
+
+    @DisplayName("@RequestMapping 애너테이션이 붙은 메서드로 요청을 처리한다.")
+    @Test
+    void handle()
+            throws Exception {
+        Reflections reflections = new Reflections("samples");
+
+        Set<Class<?>> controllers = reflections.getTypesAnnotatedWith(Controller.class);
+        Class<?> controller = controllers.stream()
+                .findAny()
+                .orElseThrow();
+
+        Method[] methods = controller.getDeclaredMethods();
+
+        Method method = Arrays.stream(methods)
+                .findAny()
+                .orElseThrow();
+
+        HandlerExecution handlerExecution = new HandlerExecution(method);
+
+        MockHttpServletRequest request = mock(MockHttpServletRequest.class);
```

Comment:
MockHttpServletRequest을 또 stub 처리하는 것은 어색한 것 같아요.
```suggestion
        HttpServletRequest request = mock(HttpServletRequest.class);
```

Reply:
저도 이 코드를 작성할 때 이게 되네? 하고 수정을 안했네요 ㅎㅎ 감사합니다 😊